### PR TITLE
[13.0][MIG] mass_mailing_resend: Migration to 13.0

### PR DESCRIPTION
**Pull request by pedrobaeza**
- Standard procedure
- Adapted models

cc @Tecnativa TT19778

### CODE REVIEW

File: mass_mailing_resend/README.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+* Go to *Email marketing > Mailings > Mass Mailings*.
```
Comment (rafaelbn) [inline mass_mailing_resend/README.rst]:
```suggestion
* Go to *Email marketing > Mailings*.
```

File: mass_mailing_resend/README.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+====================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
```
Comment (rafaelbn) [inline mass_mailing_resend/README.rst]: This module has been working in v10, v11, v12. I think is Mature.

**Review by rafaelbn — APPROVED** [review]
:+1 tested in runbot. Just minor comment in readme.

File: mass_mailing_resend/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply mass_mailing_resend/README.rst]: It's already changed on readme/CONFIGURE.rst, but I have regenerated the main README for having it already.

File: mass_mailing_resend/README.rst (hunk 2 above)
Comment (pedrobaeza) [reply mass_mailing_resend/README.rst]: OK, done

**Review by manuelcalerosolis — APPROVED** [review]
These changes are correct

**Pull request merged**